src/Configuration/Http/Requests/ConfigureRequest.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelCom\Integration\Configuration\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Validation\Rules\Enum;
+use MyParcelCom\Integration\Configuration\Properties\PropertyType;
+
+class ConfigureRequest extends FormRequest
+{
+    public function rules(): array
+    {
+        return [
+            'data.properties' => 'array',
+            'data.properties.*.name' => 'required|string',
+            'data.properties.*.type' => ['required', new Enum(PropertyType::class)],
+            'data.properties.*.description' => 'string',
+            'data.properties.*.enum' => 'array',
+            'data.properties.*.meta' => 'array',
+        ];
+    }
+}
```

Comment:
This request seems obsolete. After all, the schema is presented by the marketplace as a response, and then our API (through the backoffice) will send form values. Therefore if there is a _ConfigureRequest_, it shouldn't have any meta, enum, description, type info, but just names and values.